sdk/servicebus/service-bus/test/managementClient.spec.ts

```diff
@@ -1,112 +1,312 @@
-    };
-
-    // Simulate a disconnect being called with a non-retryable error.
-    connectionContext.connection["_connection"].idle();
-
-    // Allow rhea to clear internal setTimeouts (since we're triggering idle manually).
-    // Otherwise, it will get into a bad internal state with uncaught exceptions.
-    await delay(2000);
-
-    // peek additional messages
-    messages = await receiverClient.peek(1);
-    peekedMessageCount += messages.length;
-    peekedMessageCount.should.equal(2, "Unexpected number of peeked messages.");
-
-    refreshConnectionCalled.should.be.greaterThan(0, "refreshConnection was not called.");
+      // peek additional messages
+      deliveryId = await sender.scheduleMessage(
+        new Date("2020-04-25T12:00:00Z"),
+        TestMessage.getSample()
+      );
+      deliveryIds.push(deliveryId);
+      deliveryIds.length.should.equal(2, "Unexpected number of scheduled messages.");
+
+      refreshConnectionCalled.should.be.greaterThan(0, "refreshConnection was not called.");
+    });
   });
 
-  it("schedule can receive and settle messages after a disconnect", async function(): Promise<
-    void
-  > {
-    // Create the sender and receiver.
-    sbClient = getServiceBusClient();
-    const { receiverClient, senderClient } = await getSenderReceiverClients(
-      sbClient,
-      TestClientType.UnpartitionedQueue,
-      TestClientType.UnpartitionedQueue
-    );
-    const sender = senderClient.createSender();
-    // Send a message so we have something to peek.
-
-    const deliveryIds = [];
-    let deliveryId = await sender.scheduleMessage(
-      new Date("2020-04-25T12:00:00Z"),
-      TestMessage.getSample()
-    );
-    deliveryIds.push(deliveryId);
-
-    deliveryIds.length.should.equal(1, "Unexpected number of scheduled messages.");
-
-    const connectionContext = (receiverClient as QueueClient)["_context"].namespace;
-    const refreshConnection = connectionContext.refreshConnection;
-    let refreshConnectionCalled = 0;
-    connectionContext.refreshConnection = function(...args) {
-      refreshConnectionCalled++;
-      refreshConnection.apply(this, args);
-    };
-
-    // Simulate a disconnect being called with a non-retryable error.
-    connectionContext.connection["_connection"].idle();
-
-    // Allow rhea to clear internal setTimeouts (since we're triggering idle manually).
-    // Otherwise, it will get into a bad internal state with uncaught exceptions.
-    await delay(2000);
-
-    // peek additional messages
-    deliveryId = await sender.scheduleMessage(
-      new Date("2020-04-25T12:00:00Z"),
-      TestMessage.getSample()
-    );
-    deliveryIds.push(deliveryId);
-    deliveryIds.length.should.equal(2, "Unexpected number of scheduled messages.");
-
-    refreshConnectionCalled.should.be.greaterThan(0, "refreshConnection was not called.");
+  describe("Management operations should throw error if the Client is closed", function(): void {
```

Comment:
How are these different from the tests under `Errors after close()` in `serviceBusClient.spec.ts` ?

Reply:
To test the new errors we are adding in this PR, we need to close the client and then try to access the methods directly on the `context.managementClient`, because otherwise, what gets tested here is the errors that we throw from the receiver and sender

Reply:
Makes sense. I was reluctant to do that because the user would have to rely on receiverClient or receiver to perform the operations.
Now, updated as you suggested, please checkout 735cb8f